src/node.cc

```diff
@@ -3936,6 +3941,12 @@ static void StartNodeInstance(void* arg) {
     Isolate::Scope isolate_scope(isolate);
     HandleScope handle_scope(isolate);
     Local<Context> context = Context::New(isolate);
+
+    // CpuProfiler requires HandleScope
+    if (profile_cpu) {
+      isolate->GetCpuProfiler()->StartProfiling(String::NewFromUtf8(isolate, profile_title), true);
```

Comment:
Also, I'd probably add a comment explaining that the result is intended to be picked up by add-ons.